rosie.py

```diff
@@ -18,15 +18,15 @@ def help():
 
 
 def run():
-    import rosie
+    from rosie.ceap import classifiers
```

Comment:
Either way documentation is important. I believe each module should have at least a description at the beginning of module's `__init__.py` ([for example](http://dolphm.com/pep257-good-python-docstrings-by-example/)).